doc/ug/bazel_notes.md

```diff
@@ -272,6 +272,17 @@ test --test_tag_filters=-cw310
 See the [`.bazelrc`](https://github.com/lowRISC/opentitan/blob/master/.bazelrc) file in the OpenTitan repository for more examples.
 Additionally, for more information see the Bazel [documentation](https://bazel.build/run/bazelrc).
 
+### Site-specific `.bazelrc-site` options
+
+In addition to each user's `$HOME/.bazelrc` file and the project-level configurations in [`$REPO_TOP/.bazelrc`](https://github.com/lowRISC/opentitan/blob/master/.bazelrc), there is the option to add site-specific configuration options, by adding them to the file `$REPO_TOP/.bazelrc-site`.
+
+For groups working on shared, site-specific servers, the `bazelrc-site` file can be useful for enforcing policies such as:
+- Default locations for build artifacts (e.g., new defaults for `--user-output-base` options, outside the "$HOME" filesystem).
```

Comment:
I think `--user-output-base` should be `--output_user_root`, right? By default, I believe the `outputBase` is the hash of the workspace directory's path, so manually overriding it in a fork could break multi-workspace builds, e.g. with git worktrees. https://bazel.build/remote/output-directories